### PR TITLE
add more informative message to the checking error

### PR DESCRIPTION
**Pull request by maikia**
Adds a message for the participant of the challenge about the nature of the problem when running into 'checking_error'

partly addresses: #474

### CODE REVIEW

**Review by agramfort — APPROVED** [review]

**Comment by maikia** [comment]
@tomMoral what do you think? If you are ok please merge (it's codecov/patch which does not go through)

File: ramp-engine/ramp_engine/dispatcher.py [hunk 1]
```diff
@@ -168,6 +168,14 @@ def launch_workers(self, session):
                 worker.status = 'error'
             if worker.status == 'error':
                 set_submission_state(session, submission_id, 'checking_error')
+                stderr = ("There was a problem with sending your submission"
+                          " for training. This problem is on RAMP side"
+                          " and most likely it is not related to your"
+                          " code. If this happened for the first time"
+                          " to this submission you might"
+                          " consider submitting the same code once again"
```
Comment (tomMoral) [inline ramp-engine/ramp_engine/dispatcher.py:176]:
```suggestion
                          " consider submitting the same code once again."
                          "Else, please contact the event organizers."
```

**Review by tomMoral — APPROVED** [review]
Just an extra precision but else I am fine with it.

**Comment by tomMoral** [comment]
I do not have the merge rights on this repo so feel free to not wait for me :)

**Comment by agramfort** [comment]
thx @maikia

**Pull request merged**